package.json

```diff
@@ -176,6 +176,7 @@
     "css-loader": "^3.6.0",
     "cssnano": "^4.1.10",
     "eslint": "^7.20.0",
+    "eslint-import-resolver-alias": "^1.1.2",
```

Comment:
As mentioned above, we shouldn't need this.

Reply:
Removed